contracts/scripts/Deploy.s.sol

```diff
@@ -10,13 +10,12 @@ import {IWorldID} from "@world-id-contracts/interfaces/IWorldID.sol";
 import {IPBHEntryPoint} from "../src/interfaces/IPBHEntryPoint.sol";
 import {IEntryPoint} from "@account-abstraction/contracts/interfaces/IEntryPoint.sol";
 
-contract DeployDevnet is Script {
+contract Deploy is Script {
```

Comment:
Can we specify the network here? `WORLD_ID` is not deterministic and will need to change between networks.